src/lsp/cobol_ptree/branching_statements.ml

```diff
@@ -752,8 +757,14 @@ and pp_accept_stmt ppf = function
         ppf on_exception;
       Fmt.pf ppf "@]"
   | AcceptFromArg { item; on_exception } ->
-      Fmt.pf ppf "@[ACCEPT@;<1 2>%a@ @[FROM ARGUMENT-VALUE"
-        Fmt.(box (pp_with_loc pp_ident)) item;
+      Fmt.pf ppf "@[ACCEPT@;<1 2>%a@"
+        Fmt.(box @@ any "FROM ARGUMENT-VALUE@" ++ (pp_with_loc pp_ident)) item;
```

Comment:
There's a space missing between `@` and `"` (sorry if I forgot that in my code excerpt — below as well). Also, I don't think the parentheses are needed  around `pp_with_loc …`.

Reply:
Ah, yes indeed...